web/src/LogPane.tsx

```diff
@@ -53,6 +53,52 @@ class LogPane extends Component<LogPaneProps, LogPaneState> {
     }
   }
 
+  componentDidUpdate(prevProps: LogPaneProps) {
+    if (this.props.highlightsEnabled && this.props.modalIsOpen) {
+      document.removeEventListener(
+        "selectionchange",
+        this.handleSelectionChange
+      )
+    }
+
+    if (
+      this.props.highlightsEnabled &&
+      prevProps.modalIsOpen &&
```

Comment:
Does this logic work for the initial view? When there is no prevProps?

Reply:
I changed this around per Nick's advice below, so this piece is no longer relevant.